packages/opencensus-exporter-azure/src/azure-stats.ts

```diff
@@ -0,0 +1,302 @@
+import {
```

Comment:
Please include the license header in all the newly added files.

Reply:
Should be taken care of!